e2e_test/source_inline/kafka/shared_source.slt.serial

```diff
@@ -23,17 +23,20 @@ sleep 2s
 system ok
 internal_table.mjs --name mv_before_produce --type sourcebackfill
 ----
-0,"{""num_consumed_rows"": 0, ""state"": ""Finished""}"
-1,"{""num_consumed_rows"": 0, ""state"": ""Finished""}"
-2,"{""num_consumed_rows"": 0, ""state"": ""Finished""}"
-3,"{""num_consumed_rows"": 0, ""state"": ""Finished""}"
+0,"{""num_consumed_rows"": 0, ""state"": ""Finished"", ""target_offset"": null}"
+1,"{""num_consumed_rows"": 0, ""state"": ""Finished"", ""target_offset"": null}"
+2,"{""num_consumed_rows"": 0, ""state"": ""Finished"", ""target_offset"": null}"
+3,"{""num_consumed_rows"": 0, ""state"": ""Finished"", ""target_offset"": null}"
```

Comment:
Seems like we don't have any tests to check the output of `show jobs`?

Reply:
The DDL finishes immediately. It seems not easy to test `show jobs`.